watch-faces/sensor/activity_logging_face.c

```diff
@@ -100,6 +120,16 @@ bool activity_logging_face_loop(movement_event_t event, void *context) {
                 if (movement_get_local_date_time().unit.second == 0 && state->display_index == 0) {
                     _activity_logging_face_update_display(state);
                 }
+
+                // Handle emoticon display timing using tick counter
+                if (state->show_emoticon) {
+                    state->emoticon_tick_count++;
+                    if (state->emoticon_tick_count >= 1) {
```

Comment:
```suggestion
                    if (state->emoticon_tick_count > 1) {
```
sometimes emoticon dissapears too fast, as this is not seconds but ticks